src/engine/tsconfig.json

```diff
@@ -22,6 +22,7 @@
       "dom",
       "es5",
       "es2018"
-    ]
+    ],
+    "types": ["node"]
```

Comment:
This is needed for `process.env` to build.

Reply:
We could optimize further and simply declare JUST what we expect (for webpack). That could be nice...